packages/legend-lego/style/_code-editor.scss

```diff
@@ -15,13 +15,42 @@
  */
 
 .code-editor {
+  &__background__container {
+    background: var(--color-dark-grey-50);
+  }
+
   &__container {
     height: 100%;
     width: 100%;
     position: relative;
     background: var(--color-dark-grey-50);
   }
 
+  &__icon--text-wrap {
+    margin-bottom: 1.9rem;
+  }
+
+  &__icon--text-wrap__icon {
+    font-size: 1.8rem;
+    margin-top: 0.6rem;
```

Comment:
I think in this case, maybe it's better to have an action header bar, and have a flag to turn it off (so default to true)